Misc/NEWS.d/next/Library/2021-03-02-13-45-05.bpo-43368.t9XEkQ.rst

```diff
@@ -0,0 +1,2 @@
+Fix a regression when an empty bytestring was fetched as ``None`` instead of
+``b''`` in :mod:`sqlite3`. Patch by Mariusz Felisiak.
```

Comment:
Add a note about which PR introduced the regression. For example _"The regression was introduced by GH-24562."_